pages/learn/more/masterclass.md

```diff
@@ -0,0 +1,15 @@
+---
+title: Masterclasses
+---
+
+# {{ $names.company.lower }} Masterclasses
+
+Masterclasses are a series of self-contained, in-depth exercises designed to take you from a {{ $names.company.lower }} beginner to power user. Each masterclass focusses on providing in-depth knowledge on a specific part of the {{ $names.company.lower }} system.
```

Comment:
```suggestion
Masterclasses are a series of self-contained, in-depth exercises designed to take you from a {{ $names.company.lower }} beginner to power user. Each masterclass focuses on providing in-depth knowledge on a specific part of the {{ $names.company.lower }} system.
```

Reply:
Thanks, going to take me a while to get used to those spellings 😄